concourse/tasks/deploy-to-govuk-paas.yml

```diff
@@ -28,7 +28,7 @@ run:
       cf apply-manifest -f manifest.yml
       cf set-env $CF_APP_NAME CF_STARTUP_TIMEOUT "$CF_STARTUP_TIMEOUT"
 
-      cf scale -i "$APP_INSTANCES" $CF_APP_NAME
```

Comment:
I figured we didn't need manual scaling if we were running from the policy?
Otherwise we might move intsances up/down to the value here only to have the policy immediately change tack?

Reply:
Yeah, if we've got a scaling policy we should let that take over